egs/wsj/s5/steps/segmentation/detect_speech_activity.sh

```diff
@@ -57,6 +57,9 @@ acwt=0.3
 transform_probs_opts=""
 
 segment_padding=0.2   # Duration (in seconds) of padding added to segments 
+min_segment_dur=0   # Filter out segments that are shorter than this
```

Comment:
These options seem useful to me and I think probably make sense to include.  However, I think there is an option when you build the HMM, that controls the minimum duration of segments, and I think the documentation for this option should mention that, so that the users don't think this is the only (or primary) way to control the minimum segment duration.  Also the documentation for these options is also given below, in the usage message, so you should duplicate it there.   And please make the documentation, both here, in the usage message, and in the
python script, more explicit.  For example, mention that the unit is seconds, and that consecutive segments are merged only if they are touching.